### DISCUSSION TITLE
Best capture card?

### DISCUSSION THREAD

**Discussion by Bascht74** [awawa-dev/HyperHDR#252] [Q&A]
Hi, I have a EZCAP269 capture card, that is not optimal für HDR video. 
Is there any better solution out there? 

Did somebody tried EZCAP326 with HDR/HDMI2.0? Or is it this device the same as 320/321?

Thx for any advice...

Sebastian

**Comment by awawa-dev** [comment]
Hi
EZCAP269 connected using USB2.0 (MJPEG) or USB3.0 (YUV)?

**Comment by awawa-dev** [comment]
Here you go: https://www.hyperhdr.eu/2022/04/usb-grabbers-hdr-to-sdr-quality-test.html

**Comment by Bascht74** [comment]
Thx! I have the EZCAP269 :-(
The new LUTs-tables look very nice!!!
